scripts/runner.node.mjs

```diff
@@ -658,6 +658,7 @@ async function runTests() {
       const buildResult = await spawnBun(execPath, {
         cwd: vendorPath,
         args: ["run", "build"],
+        timeout: 60_000,
```

Comment:
What's the decision for this value in particular?

Reply:
the time can vary between target and we'll be adding more projects in the future too, don't want it to be flaky